### DISCUSSION TITLE
Can I use the controls from this library without switching the application in Metro?

### DISCUSSION THREAD

**Discussion by ZzZombo** [MahApps/MahApps.Metro#4535] [Q&A]
Hello. While I'm interested in some controls from this library, I'd not want to change the look and behavior of it too drastically. So I'm asking if the controls can be used standalone, apart from `MetroWindow`, in my regular WPF windows?

**Comment by timunie** [comment]
Hi @ZzZombo , 

I think MetroWindow is not required, except for Dialogs iirc. However you may want to create your own Styles to match your look and feel.

**Reply by ZzZombo** [reply]
Thank you for your reply. I will include the themes for the controls themselves of course, just not the window stuff.
